distributed/bokeh/scheduler_html.py

```diff
@@ -163,23 +163,49 @@ def get(self):
         self.write(result)
 
 
+class _PrometheusCollector(object):
+    def __init__(self, server, prometheus_client):
+        self.server = server
+        self.prometheus_client = prometheus_client
+
+    def collect(self):
+        yield self.prometheus_client.core.GaugeMetricFamily(
+            'scheduler_workers_total',
```

Comment:
The metric names in the PR do not follow the Prometheus naming conventions (https://prometheus.io/docs/practices/naming/):

* We need a common prefix for dask / distributed e.g. `dask_distributed_`
* The suffix `_total` must only be used for counters (i.e. only increasing). This is a gauge so we cannot not use the prefix.

Reply:
I'm going to suggest the name `dask` rather than `dask_distributed`.  I think that Prometheus will only be used with Dask's distributed scheduler.  I don't think that there is a need to add `_distributed` to the namespace.